pyproject.toml

```diff
@@ -41,19 +41,20 @@ ipython = "*"
 pydocstyle = "*"
 pylint = "*"
 pylint-django = "*"
-pylint-nautobot = "*"
+pylint-nautobot = "^0.3.1"
```

Comment:
Does pylint nautobot support nautobot v1.x?

Reply:
Yes sir! 

```
[tool.pylint-nautobot]
supported_nautobot_versions = [
    "1.6.0"
]
```

Reply:
I see it now. Thanks!